.github/workflows/vpc-deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy to GKE
+
+on:
+    - pull_request
```

Comment:
Note - this would deploy each time when a PR is opened.

Reply:
Thanks, yes, that's intended while testing. Planning to change before opening up for review.